src/App.scss

```diff
@@ -130,7 +130,6 @@ header h1 {
 	background: #ffffff;
 	display: inline-block;
 	margin: 0 auto 5% auto;
-	padding: 10px 10px 5px;
```

Comment:
as mentioned, you probably want to keep some bottom padding. This is a good situation to practice rebasing. Basically, you'll need to make the change, than rebase onto this commit and fix the commit up (do a "fixup"). Then you'd make a `push --force` which would change the history. If all of it doesn't tell you anything, we can practice this later together or you can try on your own first ;)

Reply:
[I tried to follow these instructions. ](https://blog.sebastian-daschner.com/entries/git-commit-fixup-autosquash). The fixup worked, but the rebase didn't. For rebase I tried command `git rebase --autosquash --interactive base` but this threw `fatal: invalid upstream 'base'`. 

Then I tried command `git rebase --autosquash --interactive adjustStyle`. Tjhat kind of did something, but still the fixup is shown as before in commit history.